scripts/get

```diff
@@ -46,6 +46,17 @@ initOS() {
   esac
 }
 
+# runs the given command as root (detects if we are root already)
+runAsRoot() {
+  local CMD="$*"
+  
+  if ! whoami | egrep -q '^root$'; then
```

Comment:
If I remember my bash correctly, I think the best way to do this is to check if `$EUID` is equal to 0. According to [this SO answer](https://stackoverflow.com/a/18216122), `EUID` is read only and the best way to make sure it is a root user.